src/Cafe/CafeSystem.cpp

```diff
@@ -638,7 +638,7 @@ namespace CafeSystem
 		FSCDeviceHostFS_Mount(std::string("/internal/code/").c_str(), _pathToUtf8(executablePath.parent_path()), FSC_PRIORITY_BASE);
 		std::string internalExecutablePath = "/internal/code/";
 		internalExecutablePath.append(_pathToUtf8(executablePath.filename()));
-		_pathToExecutable = internalExecutablePath;
+		_pathToExecutable = std::move(internalExecutablePath);
```

Comment:
Needlessly verbose and may trip up someone in the future who tries to use either `rpxPath` or `internalExecutablePath`. This code runs once on game launch so it's not a worthy candidate for shaving of CPU microseconds anyway.